src/cmd_line/commands/file.ts

```diff
@@ -22,11 +23,10 @@ export class FileCommand extends node.CommandBase {
         return this._arguments;
     }
 
-    execute() : void {
-        if (process.platform === 'darwin') {
-            vscode.commands.executeCommand("workbench.action.files.openFileFolder");
-        } else {
-            vscode.commands.executeCommand("workbench.actiow.files.openFile");
-        }
+    execute(): void {
+        let oriPath = vscode.window.activeTextEditor.document.uri.path;
```

Comment:
Very minor complaint, but I'm generally not a big fan of abbreviated variable names when it's hard to tell what it's abbreviating.